internal/rego/rego.go

```diff
@@ -418,20 +430,115 @@ func (r Rego) Description() string {
 	return description
 }
 
+// HasLegacyAnnotations checks whenether rego file is using legacy
+// style annotations instead of OPA Rich Metadata annotations
+func (r Rego) HasLegacyAnnotations() bool {
```

Comment:
This function name does not match what it actually does, since it does not look through the header comments or other fields. It should be named `HasMetadataAnnotations` and check that `r.annotations` is not nil. Then, the check in the `convert` command can use `if HasMetadataAnnotations { continue }`.